sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+                where
+                    exists (
+                        select
+                            1
+                        from
+                            sims.student_restrictions inactive_student_restrictions
+                        where
+                            inactive_student_restrictions.student_id = students.id
+                            and inactive_student_restrictions.is_active <> true
+                    )
+            ) as "Previous (inactive) Restrictions Flag",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLF''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (FT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLP''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (PT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''BCSL''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "BCSL Overawards Balance",
+            students.disability_status as "Permanent Disability status",
+            -- Application Personal Info (does not include Institution or Program info)
+            ''"'' || program_years.program_year_desc || ''"'' as "Program Year",
+            applications.data ->> ''studentNumber'' as "Student Number",
+            applications.data ->> ''citizenship'' as "Citizenship status",
+            applications.data ->> ''bcResident'' as "BC Residency Status",
+            applications.data ->> ''indigenousStatus'' as "Indigenous Person status",
+            applications.data ->> ''aboriginalHeritage'' as "Indigenous category code",
+            applications.data ->> ''applicationPDPPDStatus'' as "PD/PPD Application Flag",
```

Comment:
the application data for pdppd status will have, `yes | noIWantToAccessOtherFundingTypes | noIDoNotHaveADisability` 

are they directly acceptable as report values?

Reply:
@dheepak-aot  I believe, when we grommed the ticket, we decided to send/retrieve the raw data for the report, no transformation was planned as part of this ticket. correct me devs @andrewsignori-aot @andrepestana-aot @guru-aot  if  Iam wrong

Reply:
I do recollect the conversation of raw data now @ann-aot . Thanks.